package.json

```diff
@@ -34,7 +34,8 @@
     "last-run": "^1.1.0",
     "object.defaults": "^1.0.0",
     "object.reduce": "^1.0.0",
-    "undertaker-registry": "^1.0.0"
+    "undertaker-registry": "^1.0.0",
+    "fast-levenshtein": "^3.0.0"
```

Comment:
Yep, a new dep! 

I originally added this as a function to the file, but spotted that this was in the dependency tree twice and figured that it was OK to use then. Open to changing it for sure.